kodilla-testing/src/test/java/com/kodilla/testing/forum/statistics/CalculateTestSuite.java

```diff
@@ -0,0 +1,141 @@
+package com.kodilla.testing.forum.statistics;
+
+import com.kodilla.testing.forum.statistics.*;
+import org.junit.*;
+import static org.mockito.Mockito.*;
+import java.util.*;
+
+public class CalculateTestSuite {
+
+    Calculate calculator;
+    Statistics statistics;
+    List<String> userList;
+
+    @Before
+    public void before() {
+        calculator = new Calculate();
+        statistics = mock(Statistics.class);
+        userList = new ArrayList<>();
+    }
+
+    @Test(expected = ArithmeticException.class)
+    public void shouldThrowArithmeticExceptionWhenNumberOfUserIs0() {
+        //Given
+        calculator.calculateAdvStatistics(statistics);
+        //When
+        when(statistics.usersNames()).thenReturn(userList);
+        //Then
+    }
+
+    @Test
+    public void shouldReturn1WhenUsersAre100AndPostsAre1000 () {
+        //Given
+
+        for (Integer i=0 ;i <100; i++) {
+            userList.add("FooBar" + i.toString());
+        }
+        when(statistics.usersNames()).thenReturn(userList);
+        when(statistics.postsCount()).thenReturn(1000);
+        calculator.calculateAdvStatistics(statistics);
+
+        //When
+        double averagePostsPerUser =  calculator.getAveragePostsPerUser();
+        //Then
+        Assert.assertEquals(100,userList.size());
+        Assert.assertEquals(1000,statistics.postsCount());
+
+        //@TODO: 13/11/2018 Question: why  assertEquals(double, double) is depricated
```

Comment:
https://stackoverflow.com/questions/33274030/why-is-my-assertequals-deprecated-in-junit/33274179